ModCore/Commands/Main.cs

```diff
@@ -1,197 +1,226 @@
-﻿using System;
+﻿using System;
 using System.Threading.Tasks;
-using DSharpPlus;
-using DSharpPlus.CommandsNext;
-using DSharpPlus.CommandsNext.Attributes;
-using DSharpPlus.Entities;
+using DSharpPlus;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using DSharpPlus.Entities;
 using ModCore.Database;
 using ModCore.Entities;
 
-namespace ModCore.Commands
-{
-    public class Main
-    {
-        private DatabaseContextBuilder Database { get; }
-
+namespace ModCore.Commands

```

Comment:
I'm not sure why the diff is so cluttered. Did the encoding change?

Reply:
Visual Studio did warn me about encoding but I don't know what to do with it.

Thanks, I will make the changes later on